app/src/main/res/layout/activity_home.xml

```diff
@@ -6,13 +6,157 @@
     android:layout_height="match_parent"
     tools:context=".Home">
 
```

Comment:
Not the right place for this comment but it applies to every component in this file.
Give each component a meaningful name it'll be much easier for whoever has to work on this file next.

Reply:
By convention I think its camelCase